blockchain/blockdao.go

```diff
@@ -521,6 +523,11 @@ func (dao *blockDAO) putBlock(blk *Block) error {
 		batch.Put(blockNS, topHeightKey, height, "failed to put top height")
 	}
 
+	// only log this info if you enable explorer
+	if !dao.config.Explorer.Enabled {
+		return batch.Commit()
```

Comment:
If not enabled, the block itself won't be committed, neither?

Reply:
the block will still be committed.